packages/studio-be/src/core/bots/bot-service.ts

```diff
@@ -298,6 +299,11 @@ export class BotService {
     const hooks = await listDir(getBuiltinPath('hooks'), { fileFilter: '**/*.js' })
 
     for (const type of hooks) {
+      // Some hooks are only added when migrating
+      if (hookConfig?.[type.relativePath]?.ignoreCloud) {
+        continue
+      }
+
```

Comment:
What's the link between this and code editor ?

Reply:
Related to the isCloud property when creating a bot, so we don't add hooks which are not for the cloud